config/default/localmodelcache_validatingwebhook_cainjection_patch.yaml

```diff
@@ -3,6 +3,6 @@ kind: ValidatingWebhookConfiguration
 metadata:
   name: localmodelcache.serving.kserve.io
   annotations:
-    cert-manager.io/inject-ca-from: $(kserveNamespace)/serving-cert
+    service.beta.openshift.io/inject-cabundle: "true"
```

Comment:
I used this way instead of changing the original file
https://github.com/opendatahub-io/kserve/blob/release-v0.15/config/overlays/odh/kustomization.yaml#L45-L54

Reply:
That's fine too, however, all other webhooks have the annotation, so I kept the same pattern.

Reply:
We don't, I just kept the same pattern from the other webhooks.